cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml

```diff
@@ -47,14 +43,6 @@ spec:
               scheme: HTTP
             initialDelaySeconds: 180
             timeoutSeconds: 5
-          resources:
-            # keep request = limit to keep this container in guaranteed class
-            limits:
```

Comment:
It is not dead code. I'm removing this because the new addon manager will fight with the vertical addon resizer run within the heapster pod because both of them will modify the resource limit.